usr/bin/nxdk-ld

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+NXDK_STACKSIZE=65536
+
+lld-link -subsystem:windows -fixed:no -entry:XboxCRTEntry \
```

Comment:
We probably also want to call it nxdk-link to show that it's a Windows linker, not a UNIX style linker